_rules/SC1-2-2-video-captions.md

```diff
@@ -0,0 +1,127 @@
+---
+name: video captions
+description: |
+  Captions are available for audio information in non-streaming `video` elements.
+
+success_criterion:
+- 1.2.2 # Captions (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output]{#audio-output}
+- [Visual output]{#audio-output}
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains [audio](#audio).
+
+### Expectation
+
+For each test target, audio information that is not expressed visually in the video, is available through [captions](#captions).
+
+*Note*: Captions can be either embedded in the video file itself or can be made available trough a separate track.
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```

Comment:
Same as above, I propose: "This rule assumes that the video element is used to play a video (for example, not only used to display and image), and that there is a mechanism to start the video."